ProcessesApi.Tests/V1/E2ETests/Fixtures/ProcessFixture.cs

```diff
@@ -82,17 +84,51 @@ private void createProcess(string state)
             });
         }
 
+        private void createConProcess(string state)
```

Comment:
Can't you just keep the createProcess function and have a param for process name?